admin/visitor_statistic_image.php

```diff
@@ -25,9 +25,11 @@
 ##########################################################################
 */
 
+chdir('../');
 include("../_mysql.php");
```

Comment:
This ../ can not be correct because together with chdir(../) it would result in going one folder to far